lib/service_manager.dart

```diff
@@ -263,16 +264,25 @@ class ServiceExtensionManager {
       }
 
       if (!_firstFrameEventReceived) {
-        final EvalOnDartLibrary flutterLibrary = new EvalOnDartLibrary(
-          'package:flutter/src/widgets/binding.dart',
-          _service,
-        );
-
-        final InstanceRef value = await flutterLibrary.eval(
-            'WidgetsBinding.instance.debugDidSendFirstFrameEvent',
-            isAlive: null);
-        final bool didSendFirstFrameEvent =
-            value != null && value.valueAsString == 'true';
+        bool didSendFirstFrameEvent = false;
+        try {
+          final value = await _service.callServiceExtension(
```

Comment:
instead of using a try catch block, use whether the service extension is available. we have that info in VMServiceManager

Reply:
Done